website/docs/api-backends.md

```diff
@@ -17,16 +17,18 @@ specialized versions can be called for different backends. You can also create
 your own `Ops` subclasses with specialized routines for your layers, and use the
 [`set_current_ops`](#set_current_ops) function to change the default.
 
-| Backend    |        CPU         |        GPU         |        TPU        | Description                                                                                           |
-| ---------- | :----------------: | :----------------: | :---------------: | ----------------------------------------------------------------------------------------------------- |
-| `NumpyOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Execute via `numpy`, [`blis`](https://github.com/explosion/cython-blis) (optional) and custom Cython. |
-| `CupyOps`  | <i name="no"></i>  | <i name="yes"></i> | <i name="no"></i> | Execute via [`cupy`](https://cupy.chainer.org/) and custom CUDA.                                      |
+| Backend    |        CPU         |        GPU         |        TPU        | Description                                                                                                 |
+| ---------- | :----------------: | :----------------: | :---------------: | ----------------------------------------------------------------------------------------------------------- |
+| `NumpyOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Execute via `numpy`, [`blis`](https://github.com/explosion/cython-blis) (optional) and custom Cython.       |
+| `AppleOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Use AMX matrix multiplication units on Apple Silicon Macs.                                                  |
```

Comment:
For `AppleOps` and `MPSOps`, let's add a note that they're new from 9.0 onwards.

Reply:
Added a note for `AppleOps`, `MPSOps` was already in 8.1.x.